### PR TITLE
Responsive  about/gallery section & Footer

### LINKED ISSUE #40: Switch copyright to be below contact info
(responsive)

### LINKED ISSUE #44: Responsive  about section & Title moved Down
about-layout-fixes

### LINKED ISSUE #51: BUG on About Layout
BUG-aboutLayout

- [x] Fixed with Slight layout change

### PR DESCRIPTION
**Pull request by MrGDS2**
Responsive  about section & Title moved Down/Switch copyright to be below contact info

### CODE REVIEW

File: madgiraffestudios/src/App.scss [hunk 1]
```diff
@@ -73,19 +73,16 @@ body {
     }
 }
 
-@media only screen and (max-width:768px) {
+@media only screen and (max-width:1024px) {
```
Comment (estherkoko) [inline madgiraffestudios/src/App.scss:76]: why is this changed to 1024?

File: madgiraffestudios/src/components/About/About.scss [hunk 2]
```diff
@@ -13,8 +13,10 @@
  #aboutParent {
     padding-bottom: 100px;
  }
+#graphic-design{margin-top: 53px;}
+#illustrations,#photoshoots{ margin-top: 250px;}
 
-@media only screen and (max-width:768px) {
+@media only screen and (max-width:1024px) {
```
Comment (estherkoko) [inline madgiraffestudios/src/components/About/About.scss]: why is this 1024

File: madgiraffestudios/src/components/About/About.scss [hunk 3]
```diff
@@ -13,8 +13,10 @@
  #aboutParent {
     padding-bottom: 100px;
  }
+#graphic-design{margin-top: 53px;}
+#illustrations,#photoshoots{ margin-top: 250px;}
```
Comment (estherkoko) [inline madgiraffestudios/src/components/About/About.scss]: can we keep the formatting consistent with line 13-15

File: madgiraffestudios/src/components/About/About.scss [hunk 4]
```diff
@@ -29,12 +31,13 @@
         font-size: 2rem;
     }
 
+    #second {
```
Comment (estherkoko) [inline madgiraffestudios/src/components/About/About.scss:34]: why are these switched?

**Review by estherkoko — CHANGES_REQUESTED** [review]

File: madgiraffestudios/src/App.scss [hunk 5]
```diff
@@ -73,19 +73,16 @@ body {
...
     .elevator-pitch { text-align: center;}
     .emphasized-text { font-size: 2rem;}
     .main-img { display: none;}
     .navbar-nav{ display: table;}
- 
+    .booking-btn { width: auto; float: none!important}
```
Comment (estherkoko) [inline madgiraffestudios/src/App.scss:81]: why is this added to booking button?

File: madgiraffestudios/src/App.scss (hunk 1 above)
Comment (MrGDS2) [reply madgiraffestudios/src/App.scss:76]: it fixes some responsive things on some main page

File: madgiraffestudios/src/components/About/About.scss (hunk 4 above)
Comment (MrGDS2) [reply madgiraffestudios/src/components/About/About.scss:34]: #51

File: madgiraffestudios/src/App.scss (hunk 5 above)
Comment (MrGDS2) [reply madgiraffestudios/src/App.scss:81]: taking out 756 allowed me to add the button in the 1024...it changed nothing

File: madgiraffestudios/src/components/About/About.scss (hunk 2 above)
Comment (MrGDS2) [reply madgiraffestudios/src/components/About/About.scss]: Addressed in next PR

File: madgiraffestudios/src/App.scss (hunk 1 above)
Comment (MrGDS2) [reply madgiraffestudios/src/App.scss:76]: Addressed in next PR

File: madgiraffestudios/src/App.scss (hunk 5 above)
Comment (MrGDS2) [reply madgiraffestudios/src/App.scss:81]: Addressed in next PR changed to 991px

**Review by estherkoko — APPROVED** [review]
